src/api-gen/FunctionMarkdownBuilder.php

```diff
@@ -11,9 +11,13 @@ class FunctionMarkdownBuilder {
   private ?DocBlock $docblock;
 
   public function __construct(
-    string $file,
+    ?string $file = null,
```

Comment:
So the MethodMarkdownBuilder re-uses this FunctionMarkdownBuilder because Functions and Methods have the exact same YAML format, however Methods are contained within the Class files, and don't have their own YAML files. So, existing calls to FunctionMarkdownBuilder proceed with $file as normal, but MethodMarkdownBuilder constructs a FunctionMarkdownBuilder with a null $file, and instead passes along the sub-YAML string as $method. 

There is likely to be a more graceful way of doing this perhaps :)